packages/google_sign_in/google_sign_in/lib/google_sign_in.dart

```diff
@@ -157,18 +147,22 @@ class GoogleSignIn {
   /// The [hostedDomain] argument specifies a hosted domain restriction. By
   /// setting this, sign in will be restricted to accounts of the user in the
   /// specified domain. By default, the list of accounts will not be restricted.
-  GoogleSignIn({this.signInOption, this.scopes, this.hostedDomain});
+  GoogleSignIn(
+      {this.signInOption = SignInOption.standard,
+      this.scopes = const <String>[],
+      this.hostedDomain});
```

Comment:
nit: this will format a little nicer if you put a `,` after `this.hostedDomain`

Reply:
Thanks! I'll incorporate this change, and reformat, while rebasing the branch with the latest from master!